processor/resourcedetectionprocessor/README.md

```diff
@@ -290,10 +290,38 @@ processors:
     override: false
 ```
 
+### Openshift
+
+Queries the OpenShift and Kubernetes API to retrieve the following resource attributes:
+
+    * cloud.provider
+    * cloud.platform
+    * cloud.region
+    * k8s.cluster.name
+    * k8s.cluster.version
```

Comment:
Can cluster version change while the collector is running?  Or would a change in the cluster version cause node updates as well (which would restart the collector)?

Reply:
Regardless of the strategy selected, all nodes are replaced or are offline for a short time. So the collector should always be restarted.